lib/Features/FeatureFlagsRepository.php

```diff
@@ -15,4 +15,33 @@ class FeatureFlagsRepository extends Repository {
   protected function getEntityClassName() {
     return FeatureFlagEntity::class;
   }
+
+  /**
+   * @param array $data
+   * @throws \RuntimeException
+   * @throws \InvalidArgumentException
+   * @return FeatureFlagEntity
+   */
+  public function createOrUpdate(array $data = []) {
+    if (!$data['name']) {
+      throw new \InvalidArgumentException('Missing name');
+    }
+    $feature_flag = $this->findOneBy([
+      'name' => $data['name'],
+    ]);
+    if (!$feature_flag) {
+      $feature_flag = new FeatureFlagEntity($data['name']);
+      $this->persist($feature_flag);
+    }
+    if (isset($data['value'])) {
```

Comment:
If someone passes `['name' => 'xxx', 'value' => null]`, this will not save the `null`. I'd suggest using `array_key_exists`.